src/main/java/com/google/edith/servlets/Item.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+ /** Used to hold the properties of an item Entity in datastore. */
+public final class Item {
+  double price; 
+  long quantity;  
```

Comment:
Does the quantity have to be long ?

Reply:
Yea. When you retrieve the value from the datastore it's taken as a long, which means I can't convert it to an int.